build/commands/lib/gitPatcher.js

```diff
@@ -318,6 +318,9 @@ module.exports = class GitPatcher {
           this.logProgressLine(`Warning: Could not remove obsolete PatchInfo file at path ${patchInfoPath}: "${err.message}"`)
         })
       ops.push(removeOp)
+      if (!patchInfo) {
+        continue
```

Comment:
Consider putting a warning here since patched files won't get reset. Perhaps this can happen if we change patch info format, or if it (unlikely) becomes corrupted after .patch is removed

Reply:
ok, I will add a warning.